src/Components/BookCard/BookCard.test.js

```diff
@@ -1,10 +1,61 @@
 import React from 'react';
 import BookCard from './BookCard';
 import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import '@testing-library/jest-dom';
+import { combineReducers, createStore } from 'redux';
+import { Provider } from 'react-redux';
+import allReducers from '../../reducers/index.js'
 
 describe('BookCard', () => {
-  it('Should work', () => {
-    expect(true).toBe(true);
+
```

Comment:
Great unit test here, I like that you mock that functions were called when the buttons are clicked